src/libs/masks.c

```diff
@@ -1572,6 +1572,25 @@ static void _lib_masks_selection_change(dt_lib_module_t *self, int selectid, int
   lm->gui_reset = 0;
 }
 
+static gboolean _mouse_scroll_view(GtkWidget *treeview, GdkEventScroll *event, dt_lib_module_t *self)
+{
+  if (event->state & GDK_CONTROL_MASK)
+  {
+    dt_lib_masks_t *d = (dt_lib_masks_t *)self->data;
+    const gint increment = DT_PIXEL_APPLY_DPI(10.0);
+    const gint min_height = DT_PIXEL_APPLY_DPI(100.0);
+    const gint max_height = DT_PIXEL_APPLY_DPI(1000.0);
+    gint width, height;
+    gtk_widget_get_size_request (GTK_WIDGET(d->scroll_window), &width, &height);
+    height = height + increment * event->delta_y;
+    height = (height < min_height) ? min_height : (height > max_height) ? max_height : height;
+    gtk_widget_set_size_request(GTK_WIDGET(d->scroll_window), -1, (gint)height);
+    dt_conf_set_int("plugins/darkroom/masks/heightview", (gint)height);
```

Comment:
Don't we want to add this into darktableconfig.xml.in with a default value ? All conf values should be in darktableconfig.xml.in.

Reply:
I don't know - other libs with adjustable heights don't seem to use `darktableconfig.xml.in`... added just in case :)